sh/provision.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+CONFIGS_FOLDER=cluster-configs
+
+# Set AWS credentials
+export AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}"
+export AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
+
+# Generate the cluster config file from the template
+if [ ! -f ${CONFIGS_FOLDER}/${CLUSTER}.properties ]; then
+    echo "Cluster config file not found"
+else
+    /bin/sh generate_cluster_config.sh ${CLUSTER}
+fi
+
+# Create the TLS assets for the cluster
+kube-aws render credentials --generate-ca
```

Comment:
Will we use a new CA key for each cluster? We were thinking of sharing at least between publishing and delivery in the same region.